src/ir_Gree.h

```diff
@@ -28,89 +28,82 @@
 const uint8_t kGreeMaxTempC = 30;  // Celsius
 const uint8_t kGreeMinTempF = 61;  // Fahrenheit
 const uint8_t kGreeMaxTempF = 86;  // Fahrenheit
-const uint8_t kGreeTimerHalfHrOffset = 4;   // Mask 0b000x0000
-const uint8_t kGreeTimerTensHrOffset = 5;
-const uint8_t kGreeTimerTensHrSize = 2;     // Mask 0b0xx00000
 const uint16_t kGreeTimerMax = 24 * 60;
-const uint8_t kGreeTimerEnabledOffset = 7;  // Mask 0bx0000000
-// Byte[2]
-const uint8_t kGreeTimerHoursOffset = 0;
-const uint8_t kGreeTimerHoursSize = 4;  // Bits
-const uint8_t kGreeTurboOffset = 4;
-const uint8_t kGreeLightOffset = 5;
-// This might not be used. See #814
-const uint8_t kGreePower2Offset = 6;
-const uint8_t kGreeXfanOffset = 7;
-// Byte[3]
-const uint8_t kGreeTempExtraDegreeFOffset = 2;  // Mask 0b00000x00
-const uint8_t kGreeUseFahrenheitOffset = 3;     // Mask 0b0000x000
-// Byte[4]
-const uint8_t kGreeSwingSize = 4;  // Bits
-const uint8_t kGreeSwingLastPos =    0b0000;
-const uint8_t kGreeSwingAuto =       0b0001;
-const uint8_t kGreeSwingUp =         0b0010;
-const uint8_t kGreeSwingMiddleUp =   0b0011;
-const uint8_t kGreeSwingMiddle =     0b0100;
+
+const uint8_t kGreeSwingLastPos    = 0b0000;
+const uint8_t kGreeSwingAuto       = 0b0001;
+const uint8_t kGreeSwingUp         = 0b0010;
+const uint8_t kGreeSwingMiddleUp   = 0b0011;
+const uint8_t kGreeSwingMiddle     = 0b0100;
 const uint8_t kGreeSwingMiddleDown = 0b0101;
-const uint8_t kGreeSwingDown =       0b0110;
-const uint8_t kGreeSwingDownAuto =   0b0111;
+const uint8_t kGreeSwingDown       = 0b0110;
+const uint8_t kGreeSwingDownAuto   = 0b0111;
 const uint8_t kGreeSwingMiddleAuto = 0b1001;
-const uint8_t kGreeSwingUpAuto =     0b1011;
-// Byte[5]
-const uint8_t kGreeWiFiOffset = 6;       // Mask 0b0x000000
-const uint8_t kGreeIFeelOffset = 2;      // Mask 0b00000x00
-const uint8_t kGreeDisplayTempOffset = 0;
-const uint8_t kGreeDisplayTempSize = 2;  // Mask 0b000000xx
-const uint8_t kGreeDisplayTempOff =                    0b00;  // 0
-const uint8_t kGreeDisplayTempSet =                    0b01;  // 1
-const uint8_t kGreeDisplayTempInside =                 0b10;  // 2
-const uint8_t kGreeDisplayTempOutside =                0b11;  // 3
-
+const uint8_t kGreeSwingUpAuto     = 0b1011;
 
-// Legacy defines.
-#define GREE_AUTO kGreeAuto
-#define GREE_COOL kGreeCool
-#define GREE_DRY kGreeDry
-#define GREE_FAN kGreeFan
-#define GREE_HEAT kGreeHeat
-#define GREE_MIN_TEMP kGreeMinTempC
-#define GREE_MAX_TEMP kGreeMaxTempC
-#define GREE_FAN_MAX kGreeFanMax
-#define GREE_SWING_LAST_POS kGreeSwingLastPos
-#define GREE_SWING_AUTO kGreeSwingAuto
-#define GREE_SWING_UP kGreeSwingUp
-#define GREE_SWING_MIDDLE_UP kGreeSwingMiddleUp
-#define GREE_SWING_MIDDLE kGreeSwingMiddle
-#define GREE_SWING_MIDDLE_DOWN kGreeSwingMiddleDown
-#define GREE_SWING_DOWN kGreeSwingDown
-#define GREE_SWING_DOWN_AUTO kGreeSwingDownAuto
-#define GREE_SWING_MIDDLE_AUTO kGreeSwingMiddleAuto
-#define GREE_SWING_UP_AUTO kGreeSwingUpAuto
```

Comment:
Please retain these, as some folk may have used these `#define`s prior to v2.5 for their code/project etc.
(see https://github.com/crankyoldgit/IRremoteESP8266#upgrading-from-pre-v25)